src/main/java/co/rsk/federate/signing/hsm/HSMVersion.java

```diff
@@ -0,0 +1,35 @@
+package co.rsk.federate.signing.hsm;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum HSMVersion {
+    V1(1),
+    V2(2),
+    V3(3),
+    V4(4),
+    V5(5),
+    ;
+
+    private final int number;
+
+    HSMVersion(int number) {
+        this.number = number;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public static List<HSMVersion> getPowHSMVersions() {
+        return Arrays.asList(V2, V3, V4, V5);
+    }
+
+    public static boolean isPowHSM(int version) {
```

Comment:
Guess the plan is that this goes away once we do the changes to manage HSMVersion objects instead of `ints` right? We should only use the one below

Is that part of the new task definition?

Reply:
Correct, that's the plan. Added that requirement to the task.